src/index.ts

```diff
@@ -33,16 +35,57 @@ class SignificaStart extends Command {
   static args = [
     {
       name: 'type',
-      required: true,
+      required: false,
       options: (stacks as unknown) as string[],
     },
-    { name: 'name', required: true },
+    { name: 'name', required: false },
+  ]
+
+  questions = [
```

Comment:
Shouldn't be a static method like the one above?

Reply:
I was thinking about it, but reading the other comments I've decided to use like this, with the question on method call:

```js
    const type =
      args.type ||
      (
        await inquirer.prompt({
          message: 'Which stack would you like to use?',
          type: 'list',
          name: 'type',
          choices: [
            {
              name: 'Create React App',
              value: 'cra',
            },
            {
              name: 'Nextjs',
              value: 'next',
            },
            {
              name: 'Gatsby',
              value: 'gatby',
            },
          ],
        })
      ).type

    const name =
      args.name ||
      (
        await inquirer.prompt({
          message: "What is the project's name?",
          type: 'input',
          name: 'name',
          default: 'hello-world',
        })
      ).name
```
I guess it's better to understand which variable is for which question.
What about it?

Reply:
Looks good to me